Source/WebCore/rendering/RenderFlexibleBox.cpp

```diff
@@ -1013,6 +1013,11 @@ LayoutUnit RenderFlexibleBox::crossAxisScrollbarExtent() const
     return isHorizontalFlow() ? horizontalScrollbarHeight() : verticalScrollbarWidth();
 }
 
+bool RenderFlexibleBox::childHasScrollableOverflowCrossAxis(const RenderBox& child) const
```

Comment:
This name is a bit misleading as just because the flex item is a scroll container, it does not necessarily mean it has scrollable overflow (it may or may not have). -it should also read flexItemHasFoobar since what we have here is (always) a flex item.

Reply:
Ah, right changed to `flexItemIsScrollContainerCrossAxis` The flexItem portion of the name does make it inconsistent with the other helpers in the class FWIW